Makefile

```diff
@@ -1,18 +1,16 @@
-ROOT_DIR:=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
-
-gen_proto:
-	@echo "Building docker image"
-	docker build -t gen_proto -f gen.Dockerfile .
-	@echo "Running docker to generate proto files"
-	docker run -v "$(ROOT_DIR)"/src/proto-gen:/server/users/src/proto-gen gen_proto
-
-
 gen_cert:
 	@echo "Generating certificates"
 	make cert -C src
 
-gen: gen_proto gen_cert
+gen: gen_cert
 
 gen_and_run:
 	make gen
+	docker compose -f docker-compose-dev.yaml up --build
+
+clean_and_run:
```

Comment:
I like it, but I think separating it to clean, gen, and run, would be much better.